lib/media/segment_utils.js

```diff
@@ -249,11 +251,41 @@ shaka.media.SegmentUtils = class {
           audioCodecs.push(parsedESDSBox.codec);
           hasAudio = true;
         })
-        .box('ac-3', codecBoxParser)
-        .box('ec-3', codecBoxParser)
-        .box('opus', codecBoxParser)
-        .box('Opus', codecBoxParser)
-        .box('fLaC', codecBoxParser)
+        .box('ac-3', (box) => {
+          const parsedAudioSampleEntryBox =
```

Comment:
These boxes all have the same method. You should define a local helper function for it, like how we define `codecBoxParser`.

Reply:
Done!